src/fsharp/fsc.fs

```diff
@@ -248,19 +249,19 @@ let AdjustForScriptCompile(tcConfigB:TcConfigBuilder,commandLineSourceFiles,lexR
             else Path.Combine(tcConfigB.implicitIncludeDir, file)
         with _ ->
             error (Error(FSComp.SR.pathIsInvalid(file),rangeStartup)) 
-            
+
     let commandLineSourceFiles = 
         commandLineSourceFiles 
         |> List.map combineFilePath
-        
-    let allSources = ref []       
-    
+
+    let allSources = ref []
+
```

Comment:
Yes I fidget while I'm thinking.  And the extra whitespace annoys the ocd in me.